packages/ember-glimmer/lib/ember-views/target-action-support.js

```diff
@@ -0,0 +1,217 @@
+import { Mixin } from 'ember-metal/mixin';
```

Comment:
I agree with the intent, but I would rather see the implementation moved into the "real" `target-action-support.js`.  Every file we add to `packages/ember-glimmer/lib/<some other package name>/lib/` **must** be moved before we ship, and assuming the implementation here is the same as `ember-htmlbars/component` there is no reason to make our lives harder...